### DISCUSSION TITLE
请求把安装包提交到微软商店上，以方便下载更新

### DISCUSSION THREAD

**Discussion by louisliu15** [qier222/YesPlayMusic#1220] [Ideas]
现在微软商店已经允许提交未经打包的win32应用了，希望可以提交安装包到微软商店上，以方便直接搜索下载，以及之后的版本更新~

**Comment by pan93412** [comment]
cc @memorydream

**Reply by memorydream** [reply]
发布和上架的工作统一让 @qier222 来做比较好，不容易混乱

**Comment by qier222** [comment]
我研究看看

**Reply by louisliu15** [reply]
谢谢大佬

**Reply by liukenny961** [reply]
臣附议

**Comment by tylvn** [comment]
提交到 Microsoft Store 有可能会被网易云投诉侵权😂

**Reply by louisliu15** [reply]
我听说了，微软商店这么小众的地方都能被搞到下架，无语~

**Reply by tylvn** [reply]
毕竟 Hyplayer 和 LyricEase 都被下架了，第三方客户端只能小心谨慎了😂

**Comment by XuanboWangCN** [comment]
也许微软商店要交钱。。。。。
